pydatastructs/linear_data_structures/arrays.py

```diff
@@ -102,7 +105,7 @@ def __new__(cls, dtype=NoneType, *args, **kwargs):
                     if _check_type(args[0][i], dtype) is False:
                         args[0][i] = dtype(args[0][i])
                 obj._size, obj._data = len(args[0]), \
-                                        [arg for arg in args[0]]
+                                       [arg for arg in args[0]]
```

Comment:
Irrelevant changes between L9 to L108.

Reply:
Sorry, it was my editor, give me a day, Ill change them back, but some of them are changes according to PEP 8, should I change those back?

Reply:
> should I change those back?

Yes